src/examples/libpmemobj/libart/libart1/Makefile

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright 2016, FUJITSU TECHNOLOGY SOLUTIONS GMBH
+# Copyright 2016, Intel Corporation
```

Comment:
Shouldn't this refer to 2017?
How come the license checking script didn't complain?